6.analyze_features/run_analysis.sh

```diff
@@ -40,14 +40,21 @@ papermill 5a.mean_aggregated_anova_visualize.ipynb 5a.mean_aggregated_anova_visu
 papermill 5b.sum_aggregated_anova_visualize.ipynb 5b.sum_aggregated_anova_visualize.ipynb
 papermill 5c.custom_aggregated_anova_visualize.ipynb 5c.custom_aggregated_anova_visualize.ipynb
 papermill 5d.non_aggregated_anova_visualize.ipynb 5d.non_aggregated_anova_visualize.ipynb
+papermill 6.object_distance_analysis.ipynb 6.object_distance_analysis.ipynb
 
-papermill 6.object_distance_analysis.ipynb 7.object_distance_analysis.ipynb
+# change environment to R environment
```

Comment:
Recommend updating this to say op_cell_processing_env instead of R env.

Reply:
addressed